docs/creating-test-environment.md

```diff
@@ -1,216 +1,87 @@
-export IMG_REGISTRY_PORT=$(docker port op-skr-registry.localhost 5000/tcp | cut -d ":" -f2)
-```
-
-## Using a Remote Module/Image Registry
-
-In general its possible to update your registries with 2 environment variables (for the module template, and the operator image):
-
-```sh
-export MODULE_REGISTRY=your-registry-goes-here.com
-export IMG_REGISTRY=your-registry-goes-here.com
-```
 
-### Using GCP Artifact Registry
+2. Generating and Pushing the Operator Image and Charts
 
-We will be assuming you have a GCP project called `sap-kyma-jellyfish-dev`
+   Next generate and push the module image of the operator (it will use the registry referenced in the `MODULE_REGISTRY` environment variable).
 
-#### Creating your Repository
-
-We will assume you will be creating and using a Artifact Registry Repository called `operator-test`.
-
-```sh
-gcloud artifacts repositories create operator-test \
-    --repository-format=docker \
-    --location europe-west3
-```
-
-```sh
-export MODULE_REGISTRY=europe-west3-docker.pkg.dev/sap-kyma-jellyfish-dev/operator-test
-export IMG_REGISTRY=$MODULE_REGISTRY/operator-images
-```
-
-_Note: For `MODULE_REGISTRY` it is important not to define any scheme such as `https://` so that the module generation works correctly, it is appended automatically in the operators based on the environment_
-
-Now, make sure that the Read access to the repository is possible anonymously to make it work with remote clusters (e.g. in gardener)
-
-```sh
-gcloud artifacts repositories add-iam-policy-binding operator-test \
- --location=europe-west3 --member=allUsers --role=roles/artifactregistry.reader
-```
-
-#### Authenticating Locally
-
-We will assume you will be creating and using a service-account called `operator-test-sa`.
-
-Make sure to authenticate against your registry:
-
-```sh
-gcloud auth configure-docker \
-    europe-west3-docker.pkg.dev
-```
-
-#### Creating a service Account
-
-Creation of a service account is useful for productive purposes
-
-Create a Service Account (for the necessary permissions see https://cloud.google.com/iam/docs/creating-managing-service-accounts#permissions):
-
-```sh
-gcloud iam service-accounts create operator-test-sa \
-    --display-name="Operator Test Service Account"
-```
-
-```sh
-gcloud projects add-iam-policy-binding sap-kyma-jellyfish-dev \
-      --member='serviceAccount:operator-test-sa@sap-kyma-jellyfish-dev.iam.gserviceaccount.com' \
-      --role='roles/artifactregistry.reader' \
-      --role='roles/artifactregistry.writer'
-```
+    ```sh
+    make module-operator-chart module-image
+    ```
 
-Impersonate the service-account
+# 3. Install Kyma with your module
 
-```sh
-gcloud auth print-access-token --impersonate-service-account operator-test-sa@sap-kyma-jellyfish-dev.iam.gserviceaccount.com
-```
+## 3.1 Pre-requisites
 
-Verify your login:
+_Note for two cluster mode: set your `KUBECONFIG` to the SKR Cluster context_
```

Comment:
This is the wrong context I think, mandatory `kyma-system` is only in KCP

Reply:
Now looking at the guide, this is not the correct way of running single-cluster mode. Instead the module-template or the operators need to adjust their installation target. Probably the best way to achieve this is by introducing a flag into `module-manager` that install clusters not based on a secret, but instead locally against its own kubecontext. In the same way, the Kyma CR, should be created differently, namely with `spec.sync` disabled.

@adityabhatia I think this requires adjustments. Currently the only way to run in single-cluster mode is to use the secret from the cluster as access credential for itself.

Reply:
Overall I would suggest referencing https://github.com/kyma-project/lifecycle-manager/blob/main/operator/api/v1alpha1/kyma_types.go#L68-L87 to also enable sync settings into the `module-manager` that can be passed over and be initially created differently from the CLI / Scripts, or we introduce flags in both operators to differentiate on startup since there is no use case to differentiate that during runtime

Reply:
Makes sense, that way we also do not default to local installation as default, when secret was not found. I have added the requirement as part of this issue.
https://github.com/kyma-project/module-manager/issues/78